README.md

```diff
@@ -19,12 +19,25 @@ This module provides Stackdriver Debugger support for Node.js applications. [Sta
 ## Quick Start
 ```shell
 # Install with `npm` or add to your `package.json`.
-npm install --save @google/cloud-debug
+npm install --save @google-cloud/debug
+```
 
-# Require and start the agent at the top of your main script (but after '@google/cloud-trace' if you are also using it).
-require('@google/cloud-debug').start();
+```js
+// Require and start in the startup of your application:
+var debug = require('@google-cloud/debug')();
```

Comment:
Okay, this is a valid concern because github would show the README from the default branch. There are two ways to solve this:

* change the default branch of the repo to (a newly created) 0.9.x branch. If we need to do a release between this module gets renamed we could also use this branch.
* Move all the recent refactoring to a 0.10.x branch and reset the master branch to be the latest 0.9.x